tests/Maker/MakeEntityTest.php

```diff
@@ -487,6 +487,37 @@ public function getTestDetails()
             ->configureDatabase(false),
         ];
 
+        if (version_compare(PHP_VERSION, '7.4', '>=')) {
+            yield 'entity_with_filed_and_type_php_equal_or_more_than_7.4' => [MakerTestDetails::createTest(
+                $this->getMakerInstance(MakeEntity::class),
+                [
+                    // entity class name
+                    'User',
+                    // add not additional fields
+                    'name',
+                    'string',
+                    '255', // length
+                    // nullable
+                    'y',
+                    'createdAt',
+                    // use default datetime
+                    '',
+                    // nullable
+                    'y',
+                    // finish adding fields
+                    '',
+                ])
+                ->setArgumentsString('--typed')
+                ->assert(function (string $output, string $directory) {
+                    $this->assertFileExists($directory.'/src/Entity/User.php');
+
+                    $content = file_get_contents($directory.'/src/Entity/User.php');
+                    $this->assertStringContainsString('private ?string $name;', $content);
+                    $this->assertStringContainsString('private ?\DateTimeInterface $createdAt;', $content);
+                }),
+            ];
+        }
```

Comment:
I think you should remove the comments here, or at least put them in the same line as what they are referring to (because currently, it's confusing which line, below or above, they are referring to).

Reply:
```
[
  'value',  // comment
  'value', // comment
],
```
would be preferred as long as it doesn't trip `php_cs_fixer`